rollup.config.js

```diff
@@ -1,40 +1,27 @@
-// todo checkout https://github.com/alexjoverm/typescript-library-starter/blob/master/package.json
+import typescript from "rollup-plugin-typescript2";
+import { terser } from "rollup-plugin-terser";
 
-import typescript from 'rollup-plugin-typescript2'
-import peerDepsExternal from 'rollup-plugin-peer-deps-external'
-import {terser} from "rollup-plugin-terser"
-
-import pkg from './package.json'
+import pkg from "./package.json";
 
 export default {
   input: `src/index.ts`,
   watch: {
-    include: 'src/**',
+    include: "src/**",
   },
-  plugins: [
-    typescript(),
-    peerDepsExternal({
-      packageJsonPath: './package.json'
-    }),
-    terser(),
-  ],
+  plugins: [typescript(), terser()],
   output: [
     {
       file: pkg.main,
-      format: 'cjs'
+      format: "cjs",
     },
     {
       file: pkg.module,
-      format: 'es' // the preferred format
+      format: "es", // the preferred format
+    },
+    {
+      file: pkg.browser,
+      format: "iife",
+      name: "PLD",
     },
-    // {
-    //   file: pkg.browser,
-    //   format: 'iife',
-    //   name: 'PLD' // the global which can be used in a browser
-    // }
   ],
-  external: [
```

Comment:
Any reason you removed this? The intention was to have all dependencies like lodash merge be required as peer dependencies so if you also use that package in your app, it does not get bundled twice, once in the library and once in the project.

Reply:
This is needed when you are directly importing any external packages and using them.

Reasons why this is not need for the current setup.

-  No packages are directly imported and used here --> https://github.com/vladnicula/proxy-live-document/blob/master/src/index.ts
- https://www.npmjs.com/package/@rollup/plugin-node-resolve is not being used in the config. This plugin is responsible for bundling external packages into the build. Here we are using it to bundle the `node_modules` --> https://github.com/teleporthq/teleport-registry-packer/blob/development/src/bundler/index.ts#L104
- By default `devDependencies` are not packed int he bundle since we don't use them in the code directly. Even if used, if `plugin-node-resolve` is not configured rollup marks as external dependency by default.

Once, we can check the `dist/index.module.js` in local by removing `terser`. Nothing the same `esm` input comes as `output` since no `node_modules` are bundled.